### ISSUE TITLE
Final Duel Problems

### ISSUE THREAD

**Issue by imaGecko** [OpenAngelArena/oaa#1059]
1. If the losing team reaches the kill limit during the enemy teams final duel, they win the game.

2. If one team has completely left, the final duel cannot be triggered as there are no players left to duel. Thus the game cannot end.

**Comment by Trildar** [comment]
Don't know if it was an isolated occurrence, but I recently had a game where a final Duel for the Dire seemed to trigger when they were only at some 80+ points. This happening after an unsuccessful final Duel for the Radiant side.

**Comment by KarmaticNeutral** [comment]
radiant's score at beginning of dire's win duel +10 = new kill limit?

**Comment by Trildar** [comment]
No, Radiant hasn't reached 110 either when Dire's win Duel triggered. Could also have been a normal Duel with bugged messages, but there was a Duel without the final Duel messages not too long after, so that's unlikely.

**Comment by Trildar** [comment]
Oh wait, not what you were asking. I'm not sure what the kill limit was after.

**Comment by warpdragon** [comment]
Hasn't the final duel code been corrected since this thread was last commented on @chrisinajar ?
